doc/nrf/ug_bt_fast_pair.rst

```diff
@@ -85,6 +85,13 @@ The Fast Pair service implementation provides API to generate the advertising da
   Managing memory used for the advertising packets is a responsibility of the application.
   Make sure that these functions are called by the application from the cooperative context to ensure that not discoverable advertising data generation is not preempted by an Account Key write operation from a connected Fast Pair Seeker.
   Account Keys are used to generate not discoverable advertising data.
+  Special battery data can be included in not discoverable advertising packet using the Fast Pair Battery Notification extension.
+  To use this extension, you have to:
+
+  #. Call :c:func:`bt_fast_pair_battery_set` to provide battery information.
+  #. Call :c:func:`bt_fast_pair_adv_data_fill` with the appropriate Fast Pair advertising configuration to fill the advertising buffer with data.
```

Comment:
I would rather say: Set :c:member:`bt_fast_pair_adv_config.adv_battery_mode` to include the battery notification in the generated advertising payload.

Reply:
Would:  "Set :c:member:`bt_fast_pair_adv_config.adv_battery_mode` in the call to the :c:func:`bt_fast_pair_adv_data_fill` to include the battery notification in the generated advertising payload." be okay for you?

Reply:
The same configuration must be applied while getting the advertising data size. Since both functions are mentioned at the top of the paragraph, I think we may not mention them here again.